rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -52,11 +52,11 @@ object SystemProcesses {
                               TaggedContinuation],
                 dispatcher: Dispatch[Task, ListChannelWithRandom, TaggedContinuation])
     : Seq[ListChannelWithRandom] => Task[Unit] = {
-    case Seq(ListChannelWithRandom(Seq(arg, ack), rand)) =>
+    case Seq(ListChannelWithRandom(Seq(arg, ack), rand, cost)) =>
       Task(Console.err.println(prettyPrinter.buildString(arg))).flatMap { (_: Unit) =>
         space
           .produce(ack,
-                   ListChannelWithRandom(Seq(Channel(Quote(Par.defaultInstance))), rand),
+                   ListChannelWithRandom(Seq(Channel(Quote(Par.defaultInstance))), rand, cost),
```

Comment:
It seems to me that it should be empty on produce, and that rather than passing it through, we should use an explicit None.

Reply:
Applies through this whole file.

Reply:
@guardbotmk3 I decided to merge. We can sort this in the next PR. I will get in touch.